packages/components/psammead-brand/src/index.stories.jsx

```diff
@@ -18,10 +18,19 @@ storiesOf('Brand', module)
         .map(key => key.charAt(0).toUpperCase() + key.slice(1));
 
       const choice = select('Service SVG', options, 'news').toLowerCase();
-      return <Brand brandName="Default Brand Name" svg={svgs[choice]} />;
+      return (
+        <Brand
+          brandName="Default Brand Name"
+          svg={svgs[choice]}
+          link="https://bbc.com/news"
+        />
+      );
     },
     { notes },
-  );
+  )
+  .add('without link', () => (
```

Comment:
There should not be a story testing for without a link if `link` is a required proptype?